package.json

```diff
@@ -12,111 +13,119 @@
-    "@storybook/svelte": "^8.0.0-rc.2",
-    "@storybook/svelte-vite": "^8.0.0-rc.2",
-    "@storybook/test": "^8.0.0-rc.2",
-    "@storybook/theming": "^8.0.0-rc.2",
-    "@storybook/types": "^8.0.0-rc.2",
-    "@sveltejs/package": "^2.2.0",
-    "@sveltejs/vite-plugin-svelte": "^2.4.2",
-    "@tsconfig/svelte": "^5.0.0",
-    "@types/estree": "^1.0.1",
-    "auto": "^11.1.1",
-    "chromatic": "^6.19.9",
-    "concurrently": "^8.2.0",
-    "eslint": "^7.17.0",
-    "eslint-plugin-storybook": "^0.6.12",
-    "jsdom": "^22.1.0",
-    "prettier": "^3.0.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "rimraf": "^3.0.2",
-    "storybook": "^8.0.0-rc.2",
-    "svelte": "^4.2.2",
-    "svelte-check": "^3.5.0",
-    "svelte-jester": "^2.3.2",
-    "svelte-loader": "^3.1.7",
-    "typescript": "^5.1.6",
-    "vite": "^4.4.3",
-    "vitest": "^0.33.0"
+    "@auto-it/released": "^11.1.6",
+    "@storybook/addon-actions": "^8.2.1",
+    "@storybook/addon-essentials": "^8.2.1",
+    "@storybook/addon-interactions": "^8.2.1",
+    "@storybook/client-logger": "^8.2.1",
+    "@storybook/docs-tools": "^8.2.1",
+    "@storybook/eslint-config-storybook": "^4.0.0",
+    "@storybook/preview-api": "^8.2.1",
+    "@storybook/svelte": "^8.2.1",
+    "@storybook/svelte-vite": "^8.2.1",
+    "@storybook/test": "^8.2.1",
+    "@sveltejs/package": "^2.3.2",
+    "@sveltejs/vite-plugin-svelte": "4.0.0-next.3",
+    "@tsconfig/svelte": "^5.0.4",
+    "@types/estree": "^1.0.5",
+    "@types/node": "^20.14.9",
+    "@vitest/ui": "^1.6.0",
+    "auto": "^11.1.6",
+    "chromatic": "^11.5.4",
+    "concurrently": "^8.2.2",
+    "eslint": "^7.32.0",
+    "eslint-plugin-storybook": "^0.8.0",
+    "jsdom": "^24.1.0",
+    "prettier": "^3.3.2",
+    "prettier-plugin-svelte": "^3.2.5",
+    "rimraf": "^5.0.7",
+    "rollup": "^4.18.0",
+    "storybook": "^8.2.1",
+    "svelte": "5.0.0-next.155",
+    "svelte-check": "^3.8.4",
+    "tslib": "^2.6.3",
+    "type-fest": "^4.20.1",
+    "typescript": "^5.5.2",
+    "typescript-svelte-plugin": "^0.3.39",
+    "vite": "^5.3.2",
+    "vite-plugin-inspect": "^0.8.4",
+    "vite-plugin-virtual": "^0.3.0",
+    "vitest": "^1.6.0"
   },
   "peerDependencies": {
-    "@storybook/svelte": "^7.0.0 || ^8.0.0 || ^8.0.0-beta.0",
-    "@sveltejs/vite-plugin-svelte": "^2.0.0 || ^3.0.0",
-    "svelte": "^4.0.0",
-    "svelte-loader": "^3.1.2",
-    "vite": "^4.0.0 || ^5.0.0"
+    "@storybook/svelte": "^8.0.0",
+    "@sveltejs/vite-plugin-svelte": "^4.0.0-next.0 || ^4.0.0",
+    "svelte": "^5.0.0-next.117 || ^5.0.0",
+    "vite": "^5.0.0"
   },
-  "peerDependenciesMeta": {
-    "@sveltejs/vite-plugin-svelte": {
-      "optional": true
-    },
-    "svelte-loader": {
-      "optional": true
-    },
-    "vite": {
-      "optional": true
-    }
+  "packageManager": "pnpm@9.1.3+sha512.7c2ea089e1a6af306409c4fc8c4f0897bdac32b772016196c469d9428f1fe2d5a21daf8ad6512762654ac645b5d9136bb210ec9a00afa8dbc4677843ba362ecd",
```

Comment:
@xeho91 can you remind me why this hash is needed here, and we can't just use `pnpm@9.1.3`?